Java/libraries/recognizers-text-datatypes-timex-expression/src/test/java/com/microsoft/recognizers/text/datatypes/timex/expression/TestTimexResolver.java

```diff
@@ -231,6 +231,20 @@ public void dataTypesResolverDuration10seconds()
         Assert.assertNull(resolution.getValues().get(0).getStart());
         Assert.assertNull(resolution.getValues().get(0).getEnd());
     }
+    
+    @Test
+    public void dataTypesResolverDuration1hour30minutes()
+    {
+    	LocalDateTime defaultDateTime = LocalDateTime.of(1, 1, 1, 0, 0);
+        Resolution resolution = TimexResolver.resolve(new String[] { "PT1H30M" }, defaultDateTime);
```

Comment:
```suggestion
        Resolution resolution = TimexResolver.resolve(new String[] { "PT1H30M" }, null);
```

See that [resolve](https://github.com/southworks/Recognizers-Text/blob/internal/java/timex-unit-tests/apply-2455/Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexResolver.java#L19) can be null and if it's it will be `now()`

Reply:
`now()` is still not the value of a default DateTime in C# (1/1/0001 12:00 AM)

Reply:
Remove it from the tests, maybe the change should be in TimexResolver **if and only if** the tests doesn't work because of this.

Reply:
Also, what is doing in C#/Java is to use a default date to update it after. Doesn't matter the value I guess